subprojects/persistent-cache/src/main/java/org/gradle/cache/scopes/ProjectScopedCache.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.cache.scopes;
+
+import org.gradle.internal.service.scopes.Scopes;
+import org.gradle.internal.service.scopes.ServiceScope;
+
+/**
+ * Factory for creating project scoped caches. These typically live under the ~/build directory of a project.
+ */
+@ServiceScope(Scopes.Project.class)
+public interface ProjectScopedCache extends ScopedCache {
```

Comment:
Shouldn't this fail `sanityCheck` without an `@Incubating` annotation and a `since`?